ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -524,15 +469,16 @@ ColumnLayout {
 
             width: 520
             destroyOnClose: true
-            confirmButtonLabel: qsTr("Hide all collectibles minted by this community")
+            confirmButtonLabel: qsTr("Hide '%1' collectibles").arg(communityName)
             cancelBtnType: ""
             showCancelButton: true
             headerSettings.title: qsTr("Hide %1 community collectibles").arg(communityName)
             headerSettings.asset.name: communityImage
             confirmationText: qsTr("Are you sure you want to hide all community collectibles minted by %1? You will no longer see or be able to interact with these collectibles anywhere inside Status.").arg(communityName)
             onCancelButtonClicked: close()
             onConfirmButtonClicked: {
-                d.hideAllCommunityTokens(communityId)
+                root.controller.showHideGroup(communityId, false)
+                root.controller.saveSettings()
```

Comment:
Does it make sense to directly call `.saveSettings()` directly inside the `.showHideGroup()` method? AFAIK the hidden operation is ALWAYS a single action and persistent from the user action, right?

Reply:
Yeah sounds logical; will have a look :)

Reply:
Done